results/intfloat__multilingual-e5-small/MAUDGeneralEconomicAndFinancialConditionsSubjectToDisproportionateImpactModifierLegalBenchClassification.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dataset_revision": "12ca3b695563788fead87a982ad1a068284413f4",
+  "mteb_dataset_name": "MAUDGeneralEconomicAndFinancialConditionsSubjectToDisproportionateImpactModifierLegalBenchClassification",
+  "mteb_version": "1.7.7",
+  "test": {
+    "accuracy": 0.8555555555555555,
```

Comment:
I have a similar concern with scores here. Might be worth excluding MAUD for now?